airflow/jobs/scheduler_job_runner.py

```diff
@@ -1801,10 +1801,7 @@ def _handle_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
             try:
                 for ti in stuck_tis:
                     executor.revoke_task(ti=ti)
-                    self._maybe_requeue_stuck_ti(
-                        ti=ti,
-                        session=session,
-                    )
+                    self._maybe_requeue_stuck_ti(ti=ti, session=session, executor=executor)
```

Comment:
We don't do this in main, do you think that should be updated too? or is it 2.* specific? If latter, why?

Reply:
It is implemented in separate PR https://github.com/apache/airflow/pull/53435 @kaxil ..